api/nakadi-event-bus-api.yaml

```diff
@@ -333,7 +677,8 @@ paths:
         - name: batch_flush_timeout
           in: query
           description: |
-            Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached before this
+            Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached
+            before this
             time is reached the messages are immediately flushed to the client.
             A value of 0 will flush every batch events immediatly as soon as they come in.
```

Comment:
This is more difficult than that. This timeout is the timeout, after with any batch, that is being collected by the system, is closed and flushed to the client. The default value could be defined differently for different settings of `batch_limit`, for example:

  | `batch_flush_timeout` undefined | `batch_flush_timeout > 0`
------------ | ------------- | -------------
**`batch_limit` undefined** | one-by-one | flush on timeout
**`batch_limit > 0`** | flush on size | flush on size OR timeout